docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -166,15 +182,17 @@ Each update performed as a part of the transaction is replicated across multiple
 
 {{< note title="Note" >}}
 YugabyteDB currently supports optimistic concurrency control, with pessimistic concurrency control being worked on actively.
-{{</note >}}
-
+{{</ note >}}
 
-## Transaction Options
+## Transaction options
 
-You can see the various options supported by transactions by running the `\h BEGIN` statement, as shown below.
+You can see the various options supported by transactions by running the `\h BEGIN` meta-command, as shown below.
```

Comment:
You can see the various options supported by transactions by running the following `\h BEGIN` meta-command: